schema/deploy/migrations/007_create_form_changes_for_existing_project_attachments.sql

```diff
@@ -0,0 +1,13 @@
+-- Deploy cif:migrations/007_create_form_changes_for_existing_project_attachments to pg
+-- requires: tables/form_change
+-- requires: tables/project_attachment
+
+begin;
+
+  alter table cif.form_change disable trigger _100_timestamps;
```

Comment:
I think we will need to add the following triggers to be disabled as well:
- commit_form_change
- _set_previous_form_change_id
- _100_committed_changes_are_immutable